graph.c

```diff
@@ -0,0 +1,145 @@
+//idea: https://runestone.academy/ns/books/published/pythonds3/Graphs/AnAdjacencyList.html
+//Implementation of simple directed cycled weighted graph
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdbool.h>
+// #include "graph_hash_table.c"
+
+#define GRAPH_VERTICES_TABLE_SIZE 10
+
+typedef struct {
+    struct Graph_vertice_neighbor *next;
+    char *name;
+    int weight;
+} Graph_vertice_neighbor;
+
+typedef struct {
+    struct Graph_vertice_neighbor *next;
+    char* name;
+} Graph_vertice;
+
+
+typedef struct {
+    Graph_vertice *vertices_table[GRAPH_VERTICES_TABLE_SIZE];
```

Comment:
How would you get the actual size of the table?

Reply:
Nevermind, delete the counter as glib has the method to return it